src/odemis/acq/fastem.py

```diff
@@ -259,6 +288,8 @@ def acquire(roa, path, scanner, multibeam, descanner, detector, stage, ccd, beam
              a tuple that contains:
                 (model.DataArray): The acquisition data, which depends on the value of the detector.dataContent VA.
                 (Exception or None): Exception raised during the acquisition or None.
+    :param settings_obs: (SettingsObserver) VAs of all components of which some will be
```

Comment:
Please indicate that None is allowed, and what happens in such case (ie, metadata not updated).